libvast/builtins/formats/cef.cpp

```diff
@@ -6,6 +6,8 @@
 // SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
+#include "vast/to_lines.hpp"
```

Comment:
```suggestion
#include <vast/to_lines.hpp>
```

Mostly a nit; can we maybe enforce this with a clang-format rule for the builtins?

Reply:
We can add a `.clang-format` file to `libvast/builtins` with `BasedOnStyle: InheritParentConfig`, but I think `clang-format` can only changes the order of includes.

Reply:
There's an in-review patch here that would somewhat fix this, but nothing we can use now: https://reviews.llvm.org/D145843